blackbox/docs/admin/auth/hba.txt

```diff
@@ -160,3 +159,33 @@ the the ``auth.host_based`` setting, like this:
         config:
           0:
             user: crate
+
+Authenticating to Admin UI
+==========================
+
+.. hide:
+
+    cr> CREATE USER admin;
+    CREATE OK, 1 row affected (... sec)
+
+When trying to access the Admin UI of CrateDB, initially an authentication with
+the user defined with the
+:ref:`auth.trust.http_default_user <auth_trust_http_default_user>` setting
```

Comment:
you can word wrap this with a line break in the middle after `_user` and before `<auth_`